### DISCUSSION TITLE
How to install on termux

### DISCUSSION THREAD

**Discussion by Drasart135** [rapid7/metasploit-framework#19511] [Q&A]
How to install on termux

**Comment by adfoster-r7** [comment]
Metasploit installation on Termux is not supported. Refer to the following for more information:

* https://github.com/rapid7/metasploit-framework/issues/11023

**Comment by wr0x00** [comment]
r u a script kiddy? Insiders understand, and those who don't understand will also not understand forever. I can tell you that this is possible, but I cannot say, after all, any unauthorized penetration is  illegal, let alone u r a  rookie
